incubator/fairwinds-metrics/templates/controller-deployment.yaml

```diff
@@ -65,6 +73,11 @@ spec:
             - name: http
               containerPort: 2112
               protocol: TCP
+          {{- if .Values.listener.enabled }}
+            - name: listener
+              containerPort: {{ .Values.listener.port }}
```

Comment:
Probably overkill to make this configurable, but I'm almost never against more configurability.